.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+ignore = E203, E266, E501, W503, F403, F401
+max-line-length = 79
```

Comment:
This should be 88. [Source](https://black.readthedocs.io/en/stable/compatible_configs.html#flake8).

Reply:
I was trying to follow this [tutorial](https://ljvmiranda921.github.io/notebook/2018/06/21/precommits-using-black-and-flake8/) as reference when configuring `black` and `flake8` ensuring their compatibility (and that there are no conflicts).

On there, they recommended configuring `flake8` ([.flake8](https://github.com/Sage-Bionetworks/schematic/blob/develop-black-flake8/.flake8)) in the way as above. After you pointed it out, I realized that the two configurations shouldn't conflict (e.g., both `black` and `flake8` should have line lengths of 79 or 88 or anything else, but they should be the same). So to ensure that it is compatible with `black`, I've changed it back to 88.